pkg/project/auth/reviewer.go

```diff
@@ -0,0 +1,66 @@
+package auth
+
+import (
+	"fmt"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+)
+
+// Review is a list of users and groups that can access a resource
+type Review interface {
```

Comment:
Actually, you could just implement a wrapper for RAR that exposes these methods and I'd be happier (that's effectively a type cast).